src/XRP/network-clients/json-rpc-network-client.ts

```diff
@@ -42,4 +38,26 @@ export default class JsonRpcNetworkClient {
     }
     return await this.axiosInstance.request(requestOptions)
   }
+
+  public async getAccountLines(
+    account: string,
+  ): Promise<AccountLinesResponseJson> {
+    // TODO: consider an option for including the 'peer' param in the request, which limits the returned trust lines to only
+    // those shared between the two accounts. (This would have to be an argument to the method here and in i-c-client too.)
+    const accountLinesRequest = {
+      method: 'account_lines',
+      params: [
+        {
+          account: account,
+          ledger_index: 'validated',
```

Comment:
so you can either specify a specific ledger index by number, or you can use a shortcut such as `validated`, `current` or `closed`.  I imagine that we will ALWAYS want `validated` (which gets you the latest validated) because otherwise there's no guarantee that the data being returned is final.  I don't think we need an enum because this is a hard-coded constant that I can't see changing.

Reply:
https://xrpl.org/basic-data-types.html#specifying-ledgers